gateway/mw_basic_auth.go

```diff
@@ -217,27 +217,33 @@ var errUnauthorized = errors.New("Unauthorized")
 
 func (k *BasicAuthKeyIsValid) checkPassword(session *user.SessionState, plainPassword string, logger *logrus.Entry) error {
 	switch session.BasicAuthData.Hash {
-	case user.HashBCrypt:
-		if err := k.compareHashAndPassword(session.BasicAuthData.Password, plainPassword, logger); err != nil {
-			return err
-		}
 	case user.HashPlainText:
 		if session.BasicAuthData.Password != plainPassword {
 			return errUnauthorized
 		}
-	default:
+
+	case user.HashSha256:
+		fallthrough
+	case user.HashMurmur32:
+		fallthrough
+	case user.HashMurmur64:
+		fallthrough
+	case user.HashMurmur128:
```

Comment:
@titpetric wouldn't it be better to have all those types in one case instead of using `fallthrough`?

Reply:
I preferred vertical readability. Same, but go fmt has made this a bit longer :)

Reply:
I've compacted the case statement in one line